ISSUE_TEMPLATE.md

```diff
@@ -1,18 +1,60 @@
+<!-- Thank you for using Flutter!
+
+     If you are looking for support, please check out our documentation
+     or consider asking a question on Stack Overflow:
+      * https://flutter.io/
+      * https://docs.flutter.io/
+      * https://stackoverflow.com/questions/tagged/flutter?sort=frequent
+
+     If you have found a bug or if our documentation doesn't have an answer
+     to what you're looking for, then fill our the template below. Please read
+     our guide to filing a bug first:  https://flutter.io/bug-reports/
+
```

Comment:
I think the extra newlines before and after each comment block aren't necessarily a good use of space.  When I paste it into a new issue, a lot of content ends up below the fold, and removing these newlines is an easy win.

Reply:
It also increases the likelihood that the user types their content into the comment area by accident.